CHANGELOG.md

```diff
@@ -53,6 +53,27 @@
 * **auth:** New authentication system.
 The application_url must be set in the ForestLiana initializer, adding a regex CORS rule for null origin is required.
 
+## [5.4.3](https://github.com/ForestAdmin/forest-rails/compare/v5.4.2...v5.4.3) (2021-01-28)
```

Comment:
These entries are not in the right order. You have to insert them in the right place regarding their release date